version

```diff
@@ -0,0 +1 @@
+2.16.0-a.0+dev
```

Comment:
Maybe move this file to the root of `client_connection` crate? Unless we already know it would be required by another crate (but it can also be moved later if/when needed)

Reply:
done